{"uid":"e2d07bf94b0a48c8","name":"test_query_empty","fullName":"tests.unit.measurement_set.test_processing_set_xdt.TestProcessingSetXdtErrors#test_query_empty","historyId":"25c87d75b86187bf483083515b180d16","time":{"start":1752045652970,"stop":1752045652970,"duration":0},"description":"Test that query raises an exception on empty DataTree","descriptionHtml":"<p>Test that query raises an exception on empty DataTree</p>\n","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"parentSuite","value":"tests.unit.measurement_set"},{"name":"suite","value":"test_processing_set_xdt"},{"name":"subSuite","value":"TestProcessingSetXdtErrors"},{"name":"host","value":"fv-az1689-22"},{"name":"thread","value":"2784-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.unit.measurement_set.test_processing_set_xdt"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":[]},"source":"e2d07bf94b0a48c8.json","parameterValues":[]}